meson_options.txt

```diff
@@ -71,3 +71,9 @@ option(
   description : 'install zsh completion script in this directory',
   value : '',
 )
+option(
+  'warn_implicit_session_opt',
```

Comment:
I'm not delighted by this option name, but I'm not going to think too hard about better names until we're sure we even want an option.